.vscode/settings.json

```diff
@@ -51,4 +51,5 @@
     "vuex",
     "whatsnew"
   ],
+  "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
This gets rid of the bs typescript intellisense errors, I'm thinking we hit the same problem as this person https://github.com/vuejs/vue-cli/issues/4239#issuecomment-1639904958